src/services/ui/src/components/Layout/index.tsx

```diff
@@ -182,7 +181,7 @@ const ResponsiveNav = ({ isDesktop }: ResponsiveNavProps) => {
         {getLinks(!!data.user, role).map((link) => (
           <NavLink
             to={link.link}
-            target={link.link === "/faq" ? FAQ_TARGET : undefined}
+            target={link.link === "/faq" ? "_blank" : "_self"}
```

Comment:
FAQ_TARGET is not equivalent to `_blank` -- this will break the "faq tab" pattern followed by OneMAC and Mako currently, where FAQ links should open in an already-existing FAQ browser tab if a prior link has already been opened.

Reply:
@kevinhaube 

According to: https://www.w3schools.com/tags/att_a_target.asp

```html
<a target="_blank|_self|_parent|_top|framename">
```
- `_blank`: Opens the linked document in a new window or tab
-  `_self`: Opens the linked document in the same frame as it was clicked (this is default)
- `_parent`: Opens the linked document in the parent frame
- `_top`: Opens the linked document in the full body of the window
- _`framename`_: Opens the linked document in the named iframe

Does that mean FAQ_TARGET ("faq-target") is still a valid value for target?

Reply:
Correct. `FAQ_TARGET` is a valid `framename` value, so any link with the identical framename reference will open in the FAQ frame.

Reply:
Oh, I think I see. Change incoming. Let me know